_topic_maps/_topic_map.yml

```
@@ -4729,6 +4729,8 @@ Topics:
     File: virt-using-vtpm-devices
   - Name: Managing virtual machines with OpenShift Pipelines
     File: virt-managing-vms-openshift-pipelines
+  - Name: Migrating VMs in a single cluster to a different storage class
```

Maybe, for clarity? It took me a couple of reads to understand what this meant.
```suggestion
  - Name: Migrating VMs in the same cluster to a different storage class
```

---

I prefer 'single' to 'same'